### PR TITLE
Publish v6.0.0

### PR DESCRIPTION
**Pull request by shubham-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by JeffGreiner-eaton — APPROVED** [review]

File: login-workflow/package.json [hunk 1]
```diff
@@ -32,17 +32,17 @@
...
     },
     "peerDependencies": {
-        "@brightlayer-ui/react-components": "^6.1.2",
+        "@brightlayer-ui/react-components": "7.0.0",
         "@emotion/react": "^11.7.1",
         "@emotion/styled": "^11.6.0",
-        "@mui/icons-material": "^5.10.15",
-        "@mui/material": "^5.10.15",
+        "@mui/icons-material": "^6.1.4",
+        "@mui/material": "^6.1.4",
```
Comment (joebochill) [inline login-workflow/package.json:39]: New peer dependencies means this needs to be a new major version of the workflow, not a minor

File: login-workflow/CHANGELOG.md [hunk 2]
```diff
@@ -5,24 +5,42 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v5.1.0 (December 20, 2024)
+
+### Changed
+
+-   Updated styles to use the new MUI 6.
```
Comment (joebochill) [inline login-workflow/CHANGELOG.md:12]: This needs to also mention the new peer dependency requirements

File: login-workflow/README.md [hunk 3]
```diff
@@ -66,6 +66,15 @@ In short, you will need to:
 -   [Components & APIs](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/components/README.md)
 -   [Screens](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/screens/README.md)
```
Comment (joebochill) [inline login-workflow/README.md:68]: We should have a migration section for going to version 6 (if the styling stuff doesn't need any changes it should be a simple as notating the new peer dependency updates).

File: login-workflow/docs/theme.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Using the Theme in your Project
```
[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply login-workflow/docs/theme.md:1]: does this theme doc need moved to different repo or just remove?

File: login-workflow/package.json (hunk 1 above)
Comment (shubham-eaton) [reply login-workflow/package.json:39]: Changed to major version v6

File: login-workflow/README.md (hunk 3 above)
Comment (shubham-eaton) [reply login-workflow/README.md:68]: Migration guide added

File: login-workflow/docs/theme.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (shubham-eaton) [reply login-workflow/docs/theme.md]: Removed the theme doc

**Review by surajeaton — APPROVED** [review]

**Pull request merged**